scriptmodules/emulators/jzintv.sh

```diff
@@ -44,8 +52,9 @@ function build_jzintv() {
     mkdir -p jzintv/bin
     cd jzintv/src
 
+    isPlatform "rpi" && extra='EXTRA=-DPLAT_LINUX_RPI'
```

Comment:
`extra` should be declared as `local`.

Reply:
Right. Will be reflected in update to this PR.